### DISCUSSION TITLE
"Debian 13 PE" 内测版申请

### DISCUSSION THREAD

**Discussion by ghostplant** [ghostplant/web-ubuntu-mate#54] [Announcements]
## 👋 Welcome!

目前 Debian 13 PE 内测版有两个版本：Debian 13 PE (x86_64)，Debian 13 PE (Arm64)，望点星支持一下～

**Comment by ydjysngs** [comment]
I want try latest debian 13 pe(Arm64)

**Reply by ghostplant** [reply]
Sent

**Comment by yjdyamv** [comment]
我想内测x86_64版的，有无国内版，我上外网这网速不太好

**Reply by ghostplant** [reply]
百度网盘可以吗？

**Reply by yjdyamv** [reply]
可以

**Reply by ghostplant** [reply]
已邀请，地址：https://github.com/ghostplant/ubuntu-images
